### DISCUSSION TITLE
Brain dead characters can spawn duffel bag and cause item duplication

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15341] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I heard the issue from other players
sometimes all items of brain dead character will be duplicated and put inside duffel bag

### Reproduction steps

.

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in our private repo, fix coming up in a future update!
